db/migrate/20171018141030_create_wb_extra_country_data.rb

```diff
@@ -0,0 +1,11 @@
+class CreateWbExtraCountryData < ActiveRecord::Migration[5.1]
+  def change
+    create_table :wb_extra_country_data do |t|
```

Comment:
a few comments regarding this table:

- change the types of  "gdp" and "population" to integer